app/assets/stylesheets/admin/views/_contacts.scss

```diff
@@ -0,0 +1,22 @@
+.contacts {
```

Comment:
should we be using BEM for this?

So `app-view-contacts__address' etc.?

Reply:
> should we be using BEM for this?
> 
> So `app-view-contacts__address` etc.?

Oh yes, good point. 👍